packages/catalog-search/src/LearningTypeRadioFacet.jsx

```diff
@@ -28,6 +31,12 @@ const LearningTypeRadioFacet = ({ enablePathways }) => {
     } else {
       dispatch(setRefinementAction('content_type', [type]));
     }
+    if (trackingName) {
+      const learningType = type || 'any';
+      sendTrackEvent(`edx.enterprise.${trackingName}.search.learning_type_selected`, {
```

Comment:
Can you share some values for `trackingName` and `learningType `?

Reply:
Should the event name mention learner portal?

Reply:
@muhammad-ammar The `trackingName` represents the MFE, and in this case, it will be `learner_portal`, as it's passed directly from the learner portal.

The event name and payload will be structured as follows:  
- **Event Name:** `edx.enterprise.learner_portal.search.learning_type_selected`  
- **Payload:** `{ learningType: 'course' }`  

The possible learning types are:  
- course  
- program  
- learnerpathway  
- video  
- any

Reply:
Is it ok to use `any` as learning type? I am not sure what info it will give us?

Reply:
@muhammad-ammar Since the goal is to track the selected value of the "learning type" dropdown, it makes sense to store "any" to represent when the user selects that option, rather than storing an empty type or not sending an event at all...